src/lib/builders/popover/index.ts

```diff
@@ -0,0 +1,147 @@
+import { derived, writable } from 'svelte/store';
+import {
+	effect,
+	elementDerived,
+	elementMultiDerived,
+	getElementByMeltId,
+	getPlacement,
+	isBrowser,
+	styleToString,
+	sleep,
+	type PositionOptions,
+	createFocusTrap,
+	cleanup,
+	useClickOutside,
+} from '$lib/internal/helpers';
+
+type CreatePopoverArgs = {
+	positioning?: PositionOptions;
+	arrowSize?: number;
+	open?: boolean;
+};
+
+const defaults = {
+	positioning: {
+		placement: 'bottom',
+	},
+	arrowSize: 8,
+	open: false,
+} satisfies CreatePopoverArgs;
+
+export function createPopover(args?: CreatePopoverArgs) {
+	const options = { ...defaults, ...args } as CreatePopoverArgs;
+	const positioning = writable(options.positioning);
+	const arrowSize = writable(options.arrowSize);
```

Comment:
Unless we want to return these so that they are modifiable post-creation, they don't need to be writables.

The thing is, so far for controlled creations, I've been using reactive statements:
![image](https://github.com/melt-ui/melt-ui/assets/26071571/ddc4c734-71ae-4ff1-96c4-88e7a0608649)

But tbh, that runs the whole thing again, and may break something. So maybe we should return the options as stores? Maybe even a single `options` store. This is a pattern I have yet to define tbh.